config/output_files.yaml

```diff
@@ -2,3 +2,10 @@ files:
   - name: "CNV HTML report"
     input: "reports/cnv_html_report/{sample}_{type}.pathology.cnv_report.html"
     output: "results/cnv/{sample}_{type}.pathology.cnv_report.html"
+    types:
+      - T
+  - name: "General Report"
+    input: "reports/general_html_report/{sample}_{type}.general_report.html"
+    output: "results/dna/reports/{sample}_{type}.general_report.html"
+    types:
+      - T
```

Comment:
Should this be generated for normal samples too?

Reply:
Yes, fixed!